uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -949,6 +990,23 @@ define([
       }
 
       /**
+       * Loads the managees of currently logged in user
+       *
+       * @return {Promise}
+       */
+      function loadManagees() {
+        var self = this;
```

Comment:
its deeply nested callback and a bind at multiple places will make code unreadable